build-setup-nolog.sh

```diff
@@ -20,16 +20,22 @@ FASTINSTALL=false
 IS_LIBRARY=false
 SKIP_TOOLCHAIN=false
 SKIP_VALIDATE=false
+TOOLCHAIN=riscv-tools
+USE_PINNED_DEPS=true
 
 function usage
 {
-    echo "usage: build-setup.sh [ fast | --fast] [--skip-toolchain] [--library] [--skip-validate]"
-    echo "   fast: if set, pulls in a pre-compiled RISC-V toolchain for an EC2 manager instance"
-    echo "   skip-toolchain: if set, skips RISC-V toolchain handling (cloning or building)."
-    echo "                   The user must define $RISCV in their env to provide their own toolchain."
-    echo "   library: if set, initializes submodules assuming FireSim is being used"
+    echo "usage: build-setup.sh [OPTIONS] [riscv-tools | esp-tools]"
+    echo "installation types:"
+    echo "   riscv-tools: if set, builds the riscv toolchain collateral (this is the default)"
+    echo "   esp-tools: if set, builds the esp toolchain collateral used for the hwacha/gemmini accelerators"
+    echo "options:"
+    echo "   --skip-toolchain: if set, skips building extra RISC-V toolchain collateral i.e Spike,"
```

Comment:
So this is only for the extra locally built stuff now? If so, I would change the name to, e.g., `--skip-toolchain-extras` or something.

Reply:
SG